R/build-home.R

```diff
@@ -120,6 +121,40 @@
 #'     href: http://website.com
 #' ```
 #'
+#' You can completely remove the sidebar.
+#'
+#' ```
+#' home:
+#'   sidebar: FALSE
+#' ```
+#'
+#' You can provide a ready-made sidebar HTML.
+#'
+#' ```
+#' home:
+#'   sidebar:
+#'     html: path-to-sidebar.html
+#' ```
+#'
+#' You can change the order of sidebar elements (except Dev Status):
```

Comment:
Wouldn't it be better to also be better to treat Dev Status as other sections?

Reply:
Yes 😄

Reply:
Now one can. The initial sidebar has a place-holder div for badges if the sidebar structure contains the dev section.
In the tweaking code, this section is either removed (when there are no badges) or replaced.